### DISCUSSION TITLE
How to access printer instances from browser dialog

### DISCUSSION THREAD

**Discussion by mcp292** [OpenPrinting/cups#750] [Q&A]
I'm using Firefox 115.0.1 (64-bit) on Arch Linux.

I have an instance (`siccs`) that works from the commandline, but from the browser dialog I see two printers and neither print according to the instance configuration:
![image](https://github.com/OpenPrinting/cups/assets/49123398/b7e135fb-3b63-4b39-b9c4-1b20ced060fc)

**Comment by debiantriage** [comment]
This portion of the Firefox print dialog should only show local printers. Let's have `lpstat -t`.

What does
```
Print using the system dialog...
```
display?

**Comment by mcp292** [comment]
Thanks for the help!

```txt
lpstat -t
----
scheduler is running
system default destination: brother
device for brother: ipp://BRWAC50DE53C9BC.local:631/ipp/print
device for siccs: ipp://10.5.109.85
device for siccs/duplex: ipp://10.5.109.85
device for xerox: usb://Xerox/B210%20Printer?serial=HAQ704348
brother accepting requests since Wed 05 Jul 2023 10:04:30 AM MST
siccs accepting requests since Tue 11 Jul 2023 10:37:36 AM MST
siccs/duplex accepting requests since Tue 11 Jul 2023 10:37:36 AM MST
xerox accepting requests since Fri 12 Aug 2022 09:19:23 AM MST
printer brother is idle.  enabled since Wed 05 Jul 2023 10:04:30 AM MST
printer siccs is idle.  enabled since Tue 11 Jul 2023 10:37:36 AM MST
printer siccs/duplex is idle.  enabled since Tue 11 Jul 2023 10:37:36 AM MST
printer xerox is idle.  enabled since Fri 12 Aug 2022 09:19:23 AM MST
```

System dialog only shows one `siccs` printer:
![image](https://github.com/OpenPrinting/cups/assets/49123398/f1227796-f0be-4a9e-8d57-76b8e15253c6)

**Comment by zdohnal** [comment]
It looks like Firefox and GTK dialogs don't support instances right now :(  (tested on Fedora Linux 38, Firefox 115.0)...
